upgrade-install.sh

```diff
@@ -28,15 +28,15 @@ fi
 
 echo "Downloading SHASUMS256.txt..."
 curl -fsSL# -o "$tmp_dir/SHASUMS256.txt" \
-  https://github.com/homebridge/homebridge-config-ui-x/releases/download/${TARGET_VERSION}/SHASUMS256.txt
+   https://github.com/homebridge/homebridge-config-ui-x/releases/download/v${TARGET_VERSION}/SHASUMS256.txt || https://github.com/homebridge/homebridge-config-ui-x/releases/download/${TARGET_VERSION}/SHASUMS256.txt
```

Comment:
is the `||` correct here since this is not a js file?

Reply:
https://stackoverflow.com/questions/14836768/how-do-and-and-or-operators-work-in-bash